bin/spcx.ts

```diff
@@ -97,6 +97,14 @@ const app = express()
 					}
 				})
 
+				// set log-level
```

Comment:
could the log level be set on the logger itself
https://github.com/sailpoint/saas-connector-sdk-js/blob/main/lib/logger/logger.ts
also spcx is local dev only